docs/vault/overview.md

```diff
@@ -96,52 +117,52 @@ The interBTC bridge introduces multiple thresholds with different actions to ens
 
 **Secure Collateral**:
```

Comment:
If we have this as headings, we can link to this heading directly from other places. I think for the thresholds that is quite helpful.

```suggestion
#### Secure Collateral
```